src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/AnalysisParameters.groovy

```diff
@@ -6,7 +6,7 @@ package life.qbic.datamodel.dtos.imaging.parameters
  * <More detailed description - When to use, what it solves, etc.>
  *
  * @author Sven Fillinger
- * @since <versiontag>
+ * @since 1.10.0
  */
-class AnalysisParameters {
+abstract class AnalysisParameters {
```

Comment:
Why abstract?

Reply:
Because it is stated as abstract object in the ER diagram. What would you prefer?

Reply:
ah, it is not abstract, but gets extended :)

Reply:
but maybe it makes sense to make it abstract and thus avoid instances of the generalized form.

Reply:
ok, so then we will keep it like this?

Reply:
yes